src/components/UpcomingEventsList.tsx

```diff
@@ -75,6 +75,7 @@ const UpcomingEventsList = () => {
     })
 
     setOrderedUpcomingEvents(formattedEvents)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
```

Comment:
Should the `locale` variable be added as a dep to this `useEffect`?

Reply:
I don't believe we need it here